include/translatable-objects.php

```diff
@@ -12,6 +12,15 @@
  */
 class PLL_Translatable_Objects implements IteratorAggregate {
 
+	/**
+	 * Type of the main translatable object.
+	 *
+	 * @var string
+	 *
+	 * @phpstan-var non-empty-string
```

Comment:
Since it is not defined in the constructor:
```suggestion
	 * @phpstan-var non-empty-string|null
```